src/index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-content">
+      <section class="calendar">
+        <div class="calendar__day start-day"></div>
```

Comment:
You should implement the solution in the next way:
- add modifier like `calendar calendar--start-day--mon` (or any other days - tue | wed | etc...) to the whole `calendar` block (not for day-element);
- inside your styles, you should implement a general solution for any of these modifiers.